### PR TITLE
Check price None in subscription message

### PR DESCRIPTION
**Pull request by Jesus89**
Closes https://github.com/CartoDB/cartoframes/issues/1413

### CODE REVIEW

File: cartoframes/data/observatory/catalog/utils.py [hunk 1]
```diff
@@ -65,33 +65,58 @@ def _display_subscription_form_notebook(entity_id, entity_type, info, instant_li
...
-        'error': '''
-        Subscription error. Please contact to support@carto.com.
-        '''}
+    price = info.get('subscription_list_price')
+    if price is None:
+        message = '''
+        <h3>Data request</h3>
+        You are about to request the {type} <b>{id}</b>.
+        If you want to proceed, a Request will be sent to CARTO who will
+        order the data and load it into your account.
```
Comment (cmongut) [inline cartoframes/data/observatory/catalog/utils.py]: I would remove these lines as CARTO will not order the data until the customer agrees with the price.

File: cartoframes/data/observatory/catalog/utils.py [hunk 2]
```diff
@@ -65,33 +65,58 @@ def _display_subscription_form_notebook(entity_id, entity_type, info, instant_li
...
+            'error': '''
+            Request error. Please contact to support@carto.com.
+            '''}
+    else:
+        message = '''
+        <h3>Subscription contract</h3>
+        You are about to subscribe to <b>{id}</b>.
+        The cost of this {type} is <b>${price}</b>.
+        If you want to proceed, a Request will be sent to CARTO who will
+        order the data and load it into your account.
```
Comment (cmongut) [inline cartoframes/data/observatory/catalog/utils.py]: I would remove these lines as CARTO will not order the data until the customer agrees with the price.

File: cartoframes/data/observatory/catalog/utils.py (hunk 1 above)
Comment (Jesus89) [reply cartoframes/data/observatory/catalog/utils.py]: OK

File: cartoframes/data/observatory/catalog/utils.py (hunk 2 above)
Comment (Jesus89) [reply cartoframes/data/observatory/catalog/utils.py]: In this case, there is a price != None, do you still want to remove them?

Comment (cmongut) [reply cartoframes/data/observatory/catalog/utils.py]: If the price != None, we will show that message.

**Pull request merged**